### DISCUSSION TITLE
New Student Developer Pack Partner: Codédex

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#109496] [GitHub Education]
![Codedex social](https://github.com/community/community/assets/107881423/21fa252b-b999-4418-b7cf-8e694dc9aab3)

We're happy to share that [Codédex](https://www.codedex.io/) is now a part of the Student Developer Pack! Codédex is a brand new learn-to-code platform for Gen Z with courses in HTML, CSS, JavaScript, React, Git & GitHub, Command Line, and more. 

If you are a verified student through [GitHub Education](https://education.github.com/students), you can receive 6 months of [Codedex Club](https://www.codedex.io/pricing) - a premium membership - for FREE!

With a Codédex Club membership, you’ll gain access to:

- Full content catalog (Python, HTML, CSS, JavaScript, Git & GitHub, and Command Line courses and more)
- Bonus Articles & Cheat Sheets as supplemental material to our courses
- Code Challenges (think LeetCode!)
- Checkpoint and Final Projects 
- Course Certificates
- Exclusive invites to community events

To get the most out of your student benefits, start by [getting verified](https://education.github.com/students), then explore exclusive student offers from across the industry today. 

Learn more about Codédex and their GitHub Student Developer Pack offer [here](https://www.codedex.io/github-students).

**Comment by dhrumibhatia** [comment]
hi @morganersery1 i have been applying to the student developer pack but it just rejected.
can you please help me out on the same..?

**Reply by morganersery1** [reply]
@dhrumibhatia 
All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

**Comment by Yukibei** [comment]
我通过了但是为什么没有会员

**Comment by Hazuq23** [comment]
hi @morganersery1 is it possible for me to get a certificate with the student developer pack on codedex.
